src/ESPEasy.ino

```diff
@@ -248,6 +248,12 @@ void setup()
 
   timermqtt_interval = 250; // Interval for checking MQTT
   timerAwakeFromDeepSleep = millis();
+  if (Settings.UseRules && isDeepSleepEnabled())
+  {
+    String event = F("System#NoSleep=");
+    event += Settings.deepSleep;
+    rulesProcessing(event);
```

Comment:
Is the `deepSleep` setting actually the "awake" time?
If so, then we have to change it, since it is very confusing.
I have to look at it when I will be near my computer again.

Reply:
as far as I understand yes. in Misc.ino (~line 1155):
```
  addFormNumericBox( F("Sleep awake time"), F("deepsleep"), Settings.deepSleep, 0, 255);
  addUnit(F("sec"));
  addHelpButton(F("SleepMode"));
  addFormNote(F("0 = Sleep Disabled, else time awake from sleep"));
  addFormNumericBox( F("Sleep time"), F("delay"), Settings.Delay, 0, 4294);   //limited by hardware to ~1.2h
  addUnit(F("sec"));
```
`Settings.deepSleep`is the actual awake time and `Settings.Delay` is the time to sleep..  bit confusing, the other way round would probably be more self speaking.

I think this iwas already discussed in another thread (also how it should be called in the web-interface)...

So it could/should probably be refactored everywhere.